packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -10,6 +10,15 @@ const logger = createLogger({
   messageContext: 'destination-cache'
 });
 
+/**
+ * Enumerator that selects the isolation type of destination in cache.
+ * The used isolation strategy is either `Tenant` or `Tenant_User` because we want to get results for subaccount and provider tenants which rules out no-isolation or user isolation.
+ */
+export enum IsolationStrategy {
```

Comment:
@MatKuhr FYI

Reply:
Greetings from the Java world where we have: [ResilienceIsolationMode](https://github.wdf.sap.corp/MA/sdk/blob/develop/cloudplatform/resilience/src/main/java/com/sap/cloud/sdk/cloudplatform/resilience/ResilienceIsolationMode.java#L11) which is used as documented [here](https://sap.github.io/cloud-sdk/docs/java/features/resilience#multi-tenancy).

However, this is not directly what customers can configure on destination retrieval. There customers have limited options on choosing on which tenant a request is made. See the [docs on that](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/sdk-connectivity-destination-service#retrieval-strategy-options).


But I think our situation is somewhat different since we pull the tenant/user from some static context information (ThreadContext) which is [automagically created](https://sap.github.io/cloud-sdk/docs/java/features/multi-tenancy/multi-tenancy-thread-context#how-is-a-thread-context-created) by the SDK.